router/router.go

```diff
@@ -458,7 +454,7 @@ func (router *Router) emitSystemEventReceived(path string, event eventpkg.Event,
 		mimeJSON,
 		eventpkg.SystemEventReceivedData{Path: path, Event: event, Headers: ihttp.FlattenHeader(header)},
 	)
-	router.handleAsyncSubscriptions(http.MethodPost, "/", *system, nil)
+	router.handleAsyncSubscriptions(http.MethodPost, systemPathFromPath(path), *system, nil)
```

Comment:
I think this would be more idiomatic to use something like `BASEPATH` instead of the helper function call.

Reply:
The reason for helper function is that `systemPathFromPath` is compiled differently depending on build tag. That's the reason for having two implementations for those three functions